reference-implementation/lib/writable-stream.js

```diff
@@ -932,26 +947,28 @@ function WritableStreamDefaultControllerGetBackpressure(controller) {
 function WritableStreamDefaultControllerError(controller, e) {
   const stream = controller._controlledWritableStream;
 
-  assert(stream._state === 'writable' || stream._state === 'closing');
-
-  const oldState = stream._state;
+  assert(stream._state === 'writable');
 
   stream._state = 'errored';
   stream._storedError = e;
 
-  if (stream._pendingAbortRequest === undefined && stream._writer !== undefined) {
-    let readyPromiseIsPending = false;
-    if (oldState === 'writable' &&
-        WritableStreamDefaultControllerGetBackpressure(stream._writableStreamController) === true) {
-      readyPromiseIsPending = true;
+  // TODO(tyoshino): Given that writer.abort() fails immediately when the stream has been errored, shouldn't
```

Comment:
Oh, I see. That was one of the keys of the change that you made closed, etc. delayed.

So, ... but, this rationale also applies to that `WritableStreamAbort()` call after `controller.error()` returns immediately even if the WritableStream has not yet became quiescent (the stream hasn't yet reflected completion of the underlying sink's method to itself).